### DISCUSSION TITLE
Getting "Cypress has crashed" while running spec files with admin permission

### DISCUSSION THREAD

**Discussion by sarthak-verma-bcg** [cypress-io/cypress#25497] [Questions and Help]
Due to some security issue, I have to run everything as administrator. If I don’t run it as admin then I get the following error-
“This program is blocked by group policy. For more information, contact your system administrator.”

Now that I run it as admin, I'm getting the following error-
_[14116:0118/112738.949:ERROR:gpu_process_host.cc(951)] GPU process launch failed: error_code=18
[14116:0118/112738.950:FATAL:gpu_data_manager_impl_private.cc(417)] GPU process isn't usable. Goodbye._

After going through some of the threads from 2019-2020, I found a workaround by specifying an environment variable- _$env:ELECTRON_EXTRA_LAUNCH_ARGS="--disable-gpu-sandbox"_
Following that I'm able to launch cypress after this. But now that I have launched cypress, while running any spec file, the browser is crashing and I'm unable to execute any testcase.

**Comment by portyu9** [comment]
Hi @sarthak-verma-bcg 

The failure is occurring below your test code. The `gpu_process_host... error_code=18` / `GPU process isn't usable` messages are from Electron/Chromium process startup, not from a Cypress assertion or your spec.

`--disable-gpu-sandbox` is useful as a diagnostic, but I would not use it (or `--no-sandbox`) as the permanent solution. It weakens Chromium's sandbox and, in your case, only moves the crash to a later point.

On a managed Windows machine I would:

1. Have IT check the policy/endpoint-security rules affecting Cypress, Electron, and Chromium child processes. The fact that the non-admin launch is explicitly blocked by Group Policy is important evidence here.
2. Try an installed browser instead of the bundled Electron test browser:
   `npx cypress run --browser chrome --headed`
3. If Chrome or Edge is managed, check `chrome://policy` / `edge://policy`. Cypress needs `RemoteDebuggingAllowed` to be undefined or enabled.
4. Compare running the project from a normal local drive versus any mapped/network/restricted location. Electron has also produced this same GPU sandbox error for restricted filesystem locations.
5. Re-run with Cypress debugging enabled and keep the terminal output:
   `DEBUG=cypress:* npx cypress open`

There have since been Cypress reports from Windows Enterprise machines running elevated with the same `GPU process launch failed: error_code=18`, and Cypress categorized those as Electron-related.

Current Cypress also deprecates Electron as a **test browser**, so Chrome/Edge is the better direction for the actual test run. If the Cypress **Launchpad itself** crashes before you can select a browser, however, that still points to the Electron app shell interacting with the machine's enterprise policy/security configuration and needs to be addressed at that level rather than in the spec.

Please mark this answer as accepted if it helped, thank you.
